react-sdk/src/context-helpers/current-interactables-context-helper.ts

```diff
@@ -40,7 +40,7 @@ export const createInteractablesContextHelper = (
 
     return {
       description:
-        "These are the interactable components currently visible on the page that you can read and modify. Each component has an id, componentName, current props, current state,and optional schema. You can use tools to update these components on behalf of the user. Don't tell the user the ID of the components, only the name, unless they ask for it.",
+        "These are the interactable components currently visible on the page that you can read and modify. Each component has an id, componentName, current props, current state, and optional schema. You can use tools to update these components on behalf of the user. Don't tell the user the ID of the components, only the name, unless they ask for it. Some components may be selected for interaction, meaning Tambo should interact with them when responding to the next message.",
```

Comment:
Again, rewrite this whole paragraph using specific terminology and proper capitalization.

Reply:
(but I do wonder why we need to keep repeating this for every component, this should just be part of the system prompt)